Lombiq.HelpfulLibraries.OrchardCore/Contents/ContentDefinitionManagerExtensions.cs

```diff
@@ -40,6 +40,9 @@ public static class ContentDefinitionManagerExtensions
         return name;
     }
 
+    /// <summary>
+    /// Resets all content type settings to <see langword="false"/> and adds <c>TitlePart</c> to the content type.
```

Comment:
Why would we use this method?

Reply:
That is not quite obvious to me as the method isn't in use in the OSOCE solution. Going by its name, it has something to do with taxonomies, but based on what I see here best I can do is guess. If I'm missing something obvious here, please point it out. 😃

Reply:
It's not apparent to me either, please ask on Teams.